src/stores/business.ts

```diff
@@ -437,6 +456,46 @@ export const useBcrosBusiness = defineStore('bcros/business', () => {
     return false
   }
 
+  /**
+ * Creates a new comment
+ * @param url the full URL to fetch the documents
+ * @returns the fetch documents object or throws error
+ */
+  const postComment = async (businessId: string, comment: CreateCommentI) => {
+    const apiURL = useRuntimeConfig().public.legalApiURL
+    const url = `${apiURL}/businesses/${businessId}/comments`
+    return await useBcrosFetch<{ comment: CommentIF }>(url, { method: 'POST', body: { comment } })
+      .then(({ data, error }) => {
+        if (error.value || !data.value) {
+          console.warn('postComment() error - invalid response =', error?.value)
+          throw new Error(error.value.message)
+        }
+        return data?.value
+      })
+  }
+  const createCommentBusiness = async (businessId: string, comment: string): Promise<CommentIF> => {
+    const account = useBcrosAccount()
+    const accountId = account.currentAccount?.id || null
+    if (accountId === null) {
+      console.error('Unable to determine account id for filing comment')
+      return
+    }
+
+    const commentObj: CreateCommentI = {
+      comment,
+      businessId
+    }
+    // post comment to API
+    const commentRes = await postComment(businessId, commentObj)
+    // flatten and sort the comments
+    if (comments.value && comments.value.length > 0) {
+      comments.value = [commentRes.comment, ...comments.value]
+    } else {
+      comments.value = [commentRes.comment]
+    }
+    return commentRes.comment
```

Comment:
why return here ? I see up that no one is using return value.

Reply:
Removed return value